### ISSUE TITLE
Unable to close Quick Links on iPhone with VoiceOver

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#16]
On iOS with VoiceOver running, the user can tap "Open Quick Links" to make the Quick Links menu appear. However, once the menu is visible, it is impossible to close it again. The only focusable and recognizable elements are the Quick Links menu items.  

This has been tested most recently with iOS 8.4 on both an iPhone and iPad Air 2, but had been previously reported using earlier versions of iOS.  

**Expected behavior:** the "Open Quick Links" button would change to "Close Quick Links" when the menu is open, and that button should remain focusable at the top of the Quick Links menu. If the user advances focus from the last item in the menu, the tab order should cycle back to the top (to the "Close Quick Links" button).

**Comment by lcaple** [comment]
Thank you for the detailed instructions to reproduce this bug. We've addressed with issue in Pull Req #96.

Closing.
